### PR TITLE
Creation of HelpGuide.md

### PR DESCRIPTION
**Pull request by tylersnel**
Thank you for submitting a Pull Request (PR) to the Cheat Sheet Series. 

> :triangular_flag_on_post: If your PR is related to grammar/typo mistakes, please double-check the file for other mistakes in order to fix all the issues in the current cheat sheet.

Please make sure that for your contribution:

- [x] In case of a new Cheat Sheet, you have used the [Cheat Sheet template](https://github.com/OWASP/CheatSheetSeries/blob/master/templates/New_CheatSheet.md).
- [x] All the markdown files do not raise any validation policy violation, see the [policy](https://github.com/OWASP/CheatSheetSeries/actions?query=workflow%3A%22Markdown+Link+Check%22).
- [x] All the markdown files follow these [format rules](https://github.com/OWASP/CheatSheetSeries/blob/master/CONTRIBUTING.md#markdown).
- [x] All your assets are stored in the **assets** folder.
- [x] All the images used are in the **PNG** format.
- [x] Any references to websites have been formatted as `[TEXT](URL)`
- [x] You verified/tested the effectiveness of your contribution (e.g., the defensive code proposed is really an effective remediation? Please verify it works!).
- [x] The CI build of your PR pass, see the build status [here](https://github.com/OWASP/CheatSheetSeries/actions).

If your PR is related to an issue, please finish your PR text with the following line:

This PR covers issue #1411.

Thank you again for your contribution :smiley:

### CODE REVIEW

File: HelpGuide.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+**Q: Under what license are the cheat sheets available?**  
+A: Creative Commons Attribution-Share Alike 4.0 International
+
+**Q: Can I use the cheat sheets in my commercial projects?**  
+A: Yes, you can use the cheat sheets in commercial projects, provided you adhere to the terms of the open-source license under which they are released.
+
+These FAQs should help users understand the purpose, usage, and contributions related to the OWASP Cheat Sheet Series website.
+
+### Troubleshooting Guide
```
Comment (kwwall) [inline HelpGuide.md:73]: I think this section would better be presented as a numbered list or perhaps a bullet list since these are all independent steps.

**Review by kwwall — APPROVED** [review]
Only one suggestion. Otherwise, LGTM.

File: HelpGuide.md (hunk 1 above)
Comment (tylersnel) [reply HelpGuide.md:73]: Good idea I'll update

**Review by jmanico — DISMISSED** [review]
Wonderful!

File: HelpGuide.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
+# `Help Guide` Cheat Sheet
```
Comment (szh) [inline HelpGuide.md:1]:
```suggestion
# Cheat Sheet Help Guide
```

File: HelpGuide.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+## Introduction
+
+Welcome to the Cheat Sheet Help Guide. This guide is designed to help you navigate the website and make the most of the features and resources offered. The cheat sheets are packed with valuable information, and we want to ensure you have all the tools you need to use them effectively.
+
+### What's Inside This Guide
+1. Basic Navigation
+2. Features
+3. FAQ
+4. Troubleshooting Guide
+5. Feedback and Support
```
Comment (szh) [inline HelpGuide.md:12]: It'd be nice for these to be anchor links to the referenced sections

File: HelpGuide.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+### Basic Navigation
+Navigation is easy. Just click on any cheat sheet or cheat sheet series you wish to see on the left side of the site. These links won't go away as you navigate so click away without getting lost.
+![Help_Nav](https://github.com/tylersnel/CheatSheetSeries/assets/67352917/020de84c-e18f-477a-acd4-889617666308)
+
+### Features
+#### Search
+Use the search bar located at the top of the page to search for certain cheat sheets and keywords in cheat sheets. 
+![Capture2](https://github.com/tylersnel/CheatSheetSeries/assets/67352917/5af0a995-ef96-42e3-90f6-e7cc8a18cfd1)
+#### Dark Mode
+If you prefer a darker theme, there is a dark mode option for the cheat sheets. Click the dark mode button to enable dark mode, which is located next to the search bar.
```
Comment (szh) [inline HelpGuide.md:25]:
```suggestion
If you prefer a darker theme, there is a dark mode option for the cheat sheets. Click the light/dark mode button, located next to the search bar, to switch between modes.
```

File: HelpGuide.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+If you wish to visit the project's GitHub repository, click on the repository link next to the search bar.
+![Repository_Link](https://github.com/tylersnel/CheatSheetSeries/assets/67352917/8582725d-941f-4d2d-b38d-7986e2659cc6)
+
+### FAQ
+Here are some frequently asked questions (FAQs) for the OWASP Cheat Sheet Series:
+
+#### General Information
+
+**Q: What is the OWASP Cheat Sheet Series?**  
+A: The OWASP Cheat Sheet Series is a collection of concise, high-value information on specific application security topics, created by various experts in the field.
```
Comment (szh) [inline HelpGuide.md:38]:
```suggestion
A: The OWASP Cheat Sheet Series is a collection of concise, high-value informational articles on specific application security topics, created by various experts in the field.
```

File: HelpGuide.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+**Q: Who creates the cheat sheets?**  
+A: The cheat sheets are created by application security professionals who have expertise in the specific topics covered.
```
Comment (szh) [inline HelpGuide.md:41]:
```suggestion
A: The cheat sheets are created by application security professionals who have expertise in the specific topics covered. They are open source and anyone is welcome to contribute by submitting a pull request on the [GitHub repository](https://github.com/OWASP/CheatSheetSeries/).
```

File: HelpGuide.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+**Q: Can I use the cheat sheets in my projects?**  
+A: Yes, the cheat sheets are designed to be practical resources that you can use in your projects to enhance security practices.
+
+**Q: Are the cheat sheets regularly updated?**  
+A: Yes, the cheat sheets are periodically reviewed and updated to reflect the latest security practices and emerging threats.
+
+#### Contributions and Community
+
+**Q: Can I contribute to the cheat sheet series?**  
+A: Yes, contributions from the community are welcome. You can contribute by suggesting new topics, providing feedback, or updating existing cheat sheets.
```
Comment (szh) [inline HelpGuide.md:60]:
```suggestion
A: Yes, contributions from the community are welcome. You can contribute by suggesting new topics, providing feedback, or updating existing cheat sheets. Create an issue or pull request on our [GitHub repository](https://github.com/OWASP/CheatSheetSeries/).
```

**Review by szh — CHANGES_REQUESTED** [review]
Love it. I added a few editorial suggestions.

File: HelpGuide.md (hunk 1 above)
Comment (tylersnel) [reply HelpGuide.md:73]: Added bullet list to the steps

File: HelpGuide.md (hunk 3 above)
Comment (tylersnel) [reply HelpGuide.md]: Added anchor links to the sections

**Review by szh — APPROVED** [review]
Looks great! How will this page be accessed though? Do we need to add a link from the home page?

**Review by mackowski — APPROVED** [review]

**Comment by szh** [comment]
I think we need to add a link from the home page. @tylersnel can you please submit a PR for that?

**Pull request merged**